### DISCUSSION TITLE
Windows  Popup Location When Opened From ViewModel

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/Maui#1919] [Q&A]
So I am using the recommended way of opening a Popup from a ViewModel.

`var popupResult = await this.popupService.ShowPopupAsync<AlertConfig>(onPresenting: viewModel => viewModel.Load("MyTitle", "My Message", "Ok", popupSizeConstents.Small300x200));
`

When the Popup is displayed on Windows it is always located in the upper Left hand corner of the windows screen. It seems to open as expected center of display on Android and iOS even though I have set properties in the Xaml.

`    <mct:Popup.Resources>
        <ResourceDictionary>
            <Style TargetType="{x:Type local:AlertPopup}">
                <Setter Property="HorizontalOptions" Value="Center" />
                <Setter Property="VerticalOptions" Value="Center" />
                <Setter Property="CanBeDismissedByTappingOutsideOfPopup" Value="False" />
            </Style>
        </ResourceDictionary>
    </mct:Popup.Resources>`

Can anyone tell me why windows is acting this way?  Please...

**Comment by Orgbrat** [comment]
Well check the comment in "PopupService ignores the specified ViewModel instance #1733" and you will see this question answered but unsure why...

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
